customer-config/checkout-config/che.plulist.v2.json

```diff
@@ -0,0 +1,221 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://raw.githubusercontent.com/extenda/hiiretail-json-schema-registry/master/customer-config/checkout-config/che.plulist.v1.json",
+  "title": "PLU List messages",
+  "type": "array",
+  "items": {
+    "type": "object",
+    "required": [
+      "id",
+      "plu"
+    ],
+    "properties": {
+      "effectiveDate": {
+        "description": "The date when this PLU list should be applied, if present.",
+        "type": "string",
+        "format": "date",
+        "example": "2021-01-01"
+      },
+      "id": {
+        "description": "A unique identifier for the PLU list.",
+        "type": "string",
+        "minLength": 1,
+        "maxLength": 20,
+        "example": "top-5-items-plu"
+      },
+      "name": {
+        "description": "The PLU List name",
+        "type": "string",
+        "minLength": 1,
+        "maxLength": 40,
+        "example": "PLU List"
+      },
+      "description": {
+        "description": "The PLU List description",
+        "type": "string",
+        "minLength": 1,
+        "maxLength": 255,
+        "example": "The description for PLU List"
+      },
+      "deviceTypes": {
+        "description": "A list of device types that this PLUList applies to",
+        "type": "array",
+        "example": "[{'id': 'SelfCheckout'}]",
+        "items": {
+          "type": "object",
+          "properties": {
+            "id": {
+              "type": "string",
+              "minLength": 1,
+              "maxLength": 40
+            }
+          }
+          }
+      },
+      "isModifiable": {
+        "description": "If it is possible to modify the content of this PLUList further down the hierarchy",
+        "type": "boolean",
+        "example": "true"
+      },
+      "isRemovable": {
+        "description": "If it is possible to remove the content of this PLUList further down the hierarchy",
+        "type": "boolean",
+        "example": "true"
+      },
+      "content": {
+        "description": "PLU lists can contain other PLU lists and/or items.",
+        "type": "array",
+        "items": {
+          "$ref": "#/$defs/PluContentType"
+        }
+      },
+      "additionalProperties": {
+        "description": "Additional custom properties can be added via this field.",
+        "type": "array",
+        "items": {
+          "$ref": "#/$defs/AdditionalProperty"
+        }
+      }
+    }
+  },
+  "$defs": {
+    "PluContentType": {
+      "type": "object",
+      "oneOf": [
+        {
+          "$ref": "#/$defs/PluItemType"
+        },
+        {
+          "$ref": "#/$defs/PluGroupType"
+        }
+      ]
+    },
+    "PluItemType": {
+      "description": "This is an entry for a specific item, and is placed inside a PLU list.",
+      "type": "object",
+      "required": [
+        "identifier",
+        "images",
+        "type"
+      ],
+      "properties": {
+        "images": {
```

Comment:
We agreed to remove `images` from the PLU Item since it can be fetched from the Items API.
I don't think PLU Item Image should be a part of Customer Controlled Configuration, there's already a service responsible for that.

Reply:
But if the customers have not added an image url to the item data but still wants an image in the PLU List?

Reply:
I agree with Joakim. If there's an image provided here, that can be used, otherwise fetch from the Item API